config/postgres_e2e_tests.yml

```diff
@@ -0,0 +1,5 @@
+- {name: "postgres16-standard-noha", postgres_version: "16", postgres_flavor: "standard", ha_type: "none"}
```

Comment:
For the beginning, we can start with smaller test matrix. Just standard flavor is enough. We can add more later on.

Reply:
I've reduced this to only test for `standard`, Postgres 17